packages/yaml/test/test.js

```diff
@@ -59,8 +59,9 @@ test('resolves extensionless imports in conjunction with nodeResolve plugin', as
 });
 
 test('applies the optional transform method to parsed YAML', async (t) => {
-  const transform = (data) => {
-    if (Array.isArray(data)) {
+  const transform = (data, filePath) => {
+    // check that transformer is passed the right file path
+    if (Array.isArray(data) && filePath.endsWith('/array.yaml')) {
```

Comment:
```suggestion
    t.ok(filePath.endsWith('/array.yaml'), filePath);
    if (Array.isArray(data)) {
```
I'm not sure why the test is failing, but moving the file path check to a separate line should make it easier to read and debug.